src/graphql/mutations/__tests__/CreateReply.js

```diff
@@ -36,7 +36,6 @@ describe('CreateReply', () => {
       },
       { userId: 'test', appId: 'test' }
     );
-    MockDate.reset();
```

Comment:
Personally I think `MockDate.reset();` is good here because the later statements doesn't depend on mock date.